sky/clouds/service_catalog/aws_catalog.py

```diff
@@ -19,6 +19,11 @@
 
 logger = sky_logging.init_logger(__name__)
 
+# General-purpose instance with Intel Ice Lake 8375C
```

Comment:
Add: This is the latest general-purpose instance family as of Jan 2023.

Reply:
Added.